docs/releases/2.7.3.rst

```diff
@@ -0,0 +1,114 @@
+=====================================
+Welcome to the new Pootle 2.7.3 final
+=====================================
+
+*Released on 20 November 2015*
+
+Bugfix release for 2.7.2.
+
+
+Changes in Requirements
+=======================
```

Comment:
Totally agree this needs to be rethought — it ends up being a copy & paste from the previous release, and can easily result misleading.

Note there are no actual version bumps from 2.7.2 in that list. In fact, the only thing that changed is allauth (0.24.1+ required).